sycl/test/fpga_tests/fpga_ihs_float.cpp

```diff
@@ -18,7 +18,7 @@ void ap_float_cast() {
   ap_int<1 + Eout + Mout> float_cast_res =
       __spirv_ArbitraryFloatCastINTEL<1 + EA + MA, 1 + Eout + Mout>(
           A, MA, Mout, Subnorm, RndMode, RndAcc);
-  // CHECK: %{{.*}} = call spir_func i40 @_Z{{[0-9]+}}__spirv_ArbitraryFloatCastINTEL{{.*}}(i40 %{{[0-9]+}}, i32 28, i32 30, i32 0, i32 2, i32 1)
+  // CHECK: call spir_func i40 @_Z{{[0-9]+}}__spirv_ArbitraryFloatCastINTEL{{.*}}(i40 {{[%a-z0-9.]+}}, i32 28, i32 30, i32 0, i32 2, i32 1)
```

Comment:
Please, add following command:
```
// RUN: %clangxx -I %sycl_include -S -emit-llvm -fsycl -fsycl-device-only %s -Xclang -fsycl-enable-optimizations -o - | FileCheck %s
```
Last time I checked, enabling optimizations emits empty LLVM module as this code has no side effects.

Reply:
Done.

Reply:
I'm not sure, but you might want to leave original run command as well.